elodie/geolocation.py

```diff
@@ -114,6 +115,25 @@ def get_key():
     __KEY__ = config['MapQuest']['key']
     return __KEY__
 
+def get_prefer_english_names():
+    global __PREFER_ENGLISH_NAMES__
+    if __PREFER_ENGLISH_NAMES__ is not None:
+        return __PREFER_ENGLISH_NAMES__
+
+    config_file = '%s/config.ini' % constants.application_directory
+    if not path.exists(config_file):
+        return False
+
+    config = load_config()
+    if('MapQuest' not in config):
+        return False
+
+    if('prefer_english_names' not in config['MapQuest']):
+        return False
+
+    s = config['MapQuest']['prefer_english_names']
```

Comment:
Here can we just do the following instead of saving it to variable and checking against multiple versions of true?

```
__PREFER_ENGLISH_NAMES__ = bool(config['MapQuest']['prefer_english_names'])
return __PREFER_ENGLISH_NAMES__
```